index.html

```diff
@@ -10,10 +10,8 @@
     <title>Get Set FOSS!</title>
     <!-- Bootstrap CSS -->
     <link rel="stylesheet" href="assets/css/bootstrap.min.css" />
-    <!-- Icon -->
-    <link
-      href="https://cdn.lineicons.com/1.0.1/LineIcons.min.css"
-      rel="stylesheet"
```

Comment:
Why did you remove this?

Reply:
@dojutsu-user I thought this was like a failed link for the icon
I will revert this change